docs/queries/filters.md

```diff
@@ -121,44 +121,52 @@ For full details of combining filters with boolean operators, see [Combining Fil
 - `no done date`
 - `has done date`
 - `done (before|after|on) <date>`
+- `done in current (week|month|year)`
```

Comment:
On balance, I think it would be better to use `this` than `current`.

As well as being fewer characters to type (a weak argument), I feel like `done in this week` reads better than `done in current week`.
Searching through the repo in GitHub for `"this week"` there's a bunch of references to that phrase, and possibly fewer to `"current week"`.

I think that `this` also may read better than `current` once `last` and `next` are added.

Reply:
I agree that 'this' is better and more natural wording. However I'm afraid of conflict with natural dates' input as described in https://obsidian-tasks-group.github.io/obsidian-tasks/queries/filters/#dates.

Won't there be a conflict with that natural-styled input?

Reply:
Oohh - good point. That would be easy to write tests for, though, and see what happens. I think it will probably work.

Reply:
Because I think that the use of 'in' and then 'this' mean that there won't be a string that gets passed through to the natural-dates-parsing code.

Reply:
> Because I think that the use of 'in' and then 'this' mean that there won't be a string that gets passed through to the natural-dates-parsing code.

Oki, I will try that. Could you lmk the reference to the module of natural-dates-parsing code pls?

Reply:
Sure: chrono-node

https://www.npmjs.com/package/chrono-node

https://github.com/obsidian-tasks-group/obsidian-tasks/blob/8f3612552c5f8605e36fc0e6971e9b54b430c44b/package.json#L57

Reply:
> Sure: chrono-node
> 
> 
> 
> https://www.npmjs.com/package/chrono-node
> 
> 
> 
> https://github.com/obsidian-tasks-group/obsidian-tasks/blob/8f3612552c5f8605e36fc0e6971e9b54b430c44b/package.json#L57
> 
> 
> 
> 

Oh that's a separate pkg... I wonder how it takes input then. Anyways. I will try changing one DateField and see what happens =)

Reply:
I am not actually sure that chrono is used in the searches. It may be a red herring.